CHANGES.rst

```diff
@@ -12,6 +12,22 @@ Yet to be released.
 
 - [Bugfix] #15 #17 - Restore Python 2.6 compatibility. (Thanks to Marco Nenciarini!)
 
+.. attention::
+    Deprecation warnings, these symbols are slated for removal in
+    the next major release.
+
+    - ``caplog.atLevel`` for ``caplog.at_level`` and
+      ``caplog.setLevel`` for ``caplog.setLevel`` are still available
```

Comment:
This also reads kind of hard, IMHO - maybe just mention "`caplog.at_level` and `caplog.set_level` should be used instead" after "they don't follow the PEP8 convention for method names."?